liberty-maven-plugin/src/main/resources/net/wasdev/wlp/maven/plugins/MvnMessages.properties

```diff
@@ -314,9 +314,9 @@ error.install.app.dropins.directory=CWWKM2178E: The application is configured in
 error.install.app.dropins.directory.explanation=The project has conflicting configuration between the server.xml file and the pom.xml file. Install configured applications to the apps folder.
 error.install.app.dropins.directory.useraction=Remove the appDirectory parameter and run the install-apps goal again.
 
-warning.install.app.add.configuration=CWWKM2179W: The application is not configured but is configured to be installed in the apps folder. Application configuration is being added to the target server.xml file by the plug-in.
-warning.install.app.add.configuration.explanation=The application must be configured in the server.xml file to run from the apps folder.
-warning.install.app.add.configuration.useraction=Add application configuration to the source server.xml.
+warn.install.app.add.configuration=CWWKM2179W: The application is not configured but is configured to be installed in the apps folder. Application configuration is being added to the target server configuration dropins folder by the plug-in.
```

Comment:
CWWKM2179W: The application is not defined in the server configuration but the POM configuration indicates it should be installed in the apps folder. Application configuration is being added to the target server configuration dropins folder by the plug-in.